README.md

```diff
@@ -1,18 +1,27 @@
 # census
 
-Serverless deployment steps:
-<ul>  
+Type in :  `npm install -g serverless`  if you are running Serverless for the first time.
+ 
+ For basic Serverless deployment:
+ 
+1.Run CMD from local directory
 
-<li>Adjust implementation logic in handler.js
-<li>Adjust serverless.yml accordingly with function name
+2.Adjust implementation logic in handler.js
 
-</ul>
-Type the following into the command line:
-<ol>
-<li>serverless config credentials --provider aws --key *YOUR KEY* --secret *YOUR SECRET KEY* --overwrite</bold>
-<li>serverless deploy
+3.Adjust serverless.yml accordingly with function specifications
+
+4.Type the following into the command line:
+* `serverless config credentials --provider aws --key *YOUR KEY* --secret *YOUR SECRET KEY* --overwrite`
+* `serverless deploy`
+
+For **Serverless-Offline**:
+
+1. Type in: `sls offline` (Plug-in must be configured into serverless.yml file)
+
+2. Use the given ROUTE on **Postman**, eg. http://localhost:3000/2015-03-31/functions/census-dev-hello/invocations to Test appropriately
+
+3. Deploy with: `serverless deploy`
```

Comment:
testing with serverless-offline requires the `serverless deploy` command? that sounds a little fishy to me. are you sure that doesn't actually deploy?

Reply:
That simply deploys it. It isn't exactly a step that's mandatory during the local development process. I can edit it out if need be.